src/main/java/ru/sevsu/rc_manager/sound/handler/RecognizeHandler.java

```diff
@@ -1,11 +1,60 @@
 package ru.sevsu.rc_manager.sound.handler;
 
 import javax.sound.sampled.AudioInputStream;
+import java.io.*;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.annotation.PostConstruct;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.vosk.LibVosk;
+import org.vosk.LogLevel;
+import org.vosk.Model;
+import org.vosk.Recognizer;
+import ru.sevsu.rc_manager.sound.processor.SoundConverter;
+
 // Handler transcripts sound to text, if it possible
+@Component
+@Slf4j
+@RequiredArgsConstructor
 public class RecognizeHandler implements Handler {
+    private Recognizer recognizer;
+    private ObjectMapper objectMapper;
+    private final SoundConverter soundConverter;
+    @Value("${sound.recognize.model-path}")
+    private String modelPath;
+
+    @PostConstruct
+    private void init() throws IOException {
+        Model model = new Model(modelPath);
+        recognizer = new Recognizer(model, 16000);
```

Comment:
what means 16000 here? For such cases it is better to create readonly constant with an appropriate name and use it here